image_moo.php

```diff
@@ -201,13 +201,27 @@ public function check_gd()
 		}
 	}
 
+	//----------------------------------------------------------------------------------------------------------
+	// checks that we have an GD image loaded
+	//---
+
+	function _is_gd_image( $image ) {
+	    if ( is_resource( $image ) && 'gd' === get_resource_type( $image )
```

Comment:
Cleaner to do 
return ( (is_resource($image) &&  get_resource_type( $image ) === 'gd') || (is_object( $image ) && $image instanceof GdImage) );